sqlx-postgres/src/connection/describe.rs

```diff
@@ -492,13 +488,25 @@ fn visit_plan(plan: &Plan, outputs: &[String], nullables: &mut Vec<Option<bool>>
 }
 
 #[derive(serde::Deserialize)]
+#[serde(untagged)]
 enum Explain {
     /// {"Plan": ...} -- returned for most statements
-    Plan(Plan),
+    QueryPlan(QueryPlan),
     /// The string "Utility Statement" -- returned for
     /// a CALL statement
     #[serde(rename = "Utility Statement")]
-    UtilityStatement,
+    UtilityStatement(String),
```

Comment:
It appears that the `#[serde(rename = "Utility Statement")]` attribute is no longer necessary, as the `Explain` enum is now untagged.